src/data/types.rs

```diff
@@ -481,7 +481,7 @@ pub(crate) mod tests {
 
     #[test]
     fn test_big_one() {
-        assert_decl_display("struct { int i; } S;", "struct { int i; } S;");
+        assert_decl_display("struct { int i; } S;", "struct {    int i;\n} S;");
```

Comment:
This is still not correct, it will look like this when printed:
```c
struct {    int i;
} S;
```
Instead it should be
```c
struct {
    int i;
} S;
```